components/table-of-contents/src/Section/test.js

```diff
@@ -1,35 +1,71 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 
-import { Entry } from '.';
+import Section, { Entry } from '.';
 
-describe('TableOfContents Section Entry', () => {
-  let wrapper;
-  const addContent = jest.fn();
-  const removeContent = jest.fn();
-
-  it('renders without crashing', () => {
-    wrapper = shallow(
-      <Entry id="id" title="Title" toc={{ addContent, removeContent }}>
-        Section text here
-      </Entry>,
+let wrapper;
```

Comment:
I'd prefer this to not be here, and for every instance where wrapper is used below to include the `let`